tutorials/io/encrypting_save_games.rst

```diff
@@ -0,0 +1,54 @@
+.. _doc_encrypting_save_games:
+
+Encrypting save games
+=====================
+
+The class :ref:`File <class_File>` can open a file at a
```

Comment:
hmmm I believe `File` in 3.x is now `FileAccess` in 4.x

Reply:
Yeah that's what was throwing sphinx off.